From d4f44de579d28020b0127c87023b8066562792fc Mon Sep 17 00:00:00 2001
From: Hari Nagalla <hnagalla@ti.com>
Date: Tue, 11 Feb 2025 14:25:26 -0600
Subject: [PATCH 58/68] FROMLIST: arm64: dts: ti: k3-am62a7-sk: Reserve
 main_timer2 for C7x DSP

C7x DSP uses main_timer2, so mark it as reserved in linux DT.

Link: https://lore.kernel.org/linux-devicetree/20250206235200.3128163-1-jm@ti.com/
Signed-off-by: Hari Nagalla <hnagalla@ti.com>
Signed-off-by: Judith Mendez <jm@ti.com>
---
 arch/arm64/boot/dts/ti/k3-am62a7-sk.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/ti/k3-am62a7-sk.dts b/arch/arm64/boot/dts/ti/k3-am62a7-sk.dts
index 605079519026..c6764a62e063 100644
--- a/arch/arm64/boot/dts/ti/k3-am62a7-sk.dts
+++ b/arch/arm64/boot/dts/ti/k3-am62a7-sk.dts
@@ -685,6 +685,11 @@ &main_uart1 {
 	status = "reserved";
 };
 
+/* main_timer2 is used by C7x DSP */
+&main_timer2 {
+	status = "reserved";
+};
+
 &usbss0 {
 	status = "okay";
 	ti,vbus-divider;
-- 
2.47.2

